addons/udes_stock_picking_batch/models/stock_move_line.py

```diff
@@ -62,7 +62,7 @@ def _prepare_task_info(self):
             task["type"] = "product"
             task["pick_quantity"] = sum(self.mapped("product_qty"))
             task["initial_demand"] = sum(self.move_id.mapped("u_uom_initial_demand"))
-            task["quant_ids"] = self.get_quants().get_info(level=2)
+            task["quant_ids"] = self.get_quants().get_info(level=2, extra_fields={"decimal_places"})
```

Comment:
I think decimal places is quite important to have on get_info of uom.uom so maybe better to add in get_info_fields and you dont need to specify in extra_fields